pontus/week4/app/list-store.js

```diff
@@ -0,0 +1,51 @@
+var Constants = require('./constants');
+var BaseStore = require('./base');
+var Dispatcher = require('./dispatcher');
+
+var _list = [];
+
+// read
+let listStore = Object.assign({}, BaseStore, {
+	getAll: function() {
+		return _list;
+	},
+
+	get: function(id) {
+		return Object.assign({}, _list.find(function(l) { return l.id == id }));
+	},
+
+	getListProperties() { 
```

Comment:
Not sure why you are using this method. The return is identical to `getAll()` except it returns a copy of the array, maybe that's the reason? If so, I believe `getAll()` should be sufficient and you could `_list.slice()` since you use `getAll()` to `setState()`.